shared/desktop/renderer/index.js

```diff
@@ -53,6 +54,57 @@ function setupAvatar() {
   initAvatarLoad(loadUserImageMap, loadTeamImageMap)
 }
 
+var inactiveTimeoutID
+var activeTimeoutID
+
+// 5 minutes after being active, consider us inactive after
+// an additional minute of no input
+// for the purpose of marking chat messages read
+function setTimeoutListeners() {
+  window.addEventListener('mousemove', resetTimer, true)
+  window.addEventListener('keypress', resetTimer, true)
+
+  startInactiveTimer()
+}
+
+function startActiveTimer() {
+  if (!isMobile) {
+    // wait 5 minutes before adding listeners
+    activeTimeoutID = window.setTimeout(goListening, 300000)
+  }
+}
+
+function startInactiveTimer() {
+  // wait 1 minute before calling goInactive
+  inactiveTimeoutID = window.setTimeout(goInactive, 60000)
+}
+
+function resetTimer(e) {
+  window.clearTimeout(inactiveTimeoutID)
+  goActive()
+}
+
+function goInactive() {
+  console.log('Going inactive due to 1 minute of no input')
+  window.clearTimeout(inactiveTimeoutID)
+  setupStore().dispatch(changedFocus(false))
```

Comment:
we need a separate concept and not overload focus. we need an 'userActive' or something and the notifications logic should be focus && userActive